docs/organizations/using-a-coding-standard.md

```diff
@@ -1,20 +1,13 @@
 # Using a coding standard
 
 !!! info "This is a beta feature"
-    This is a new Codacy feature and <span class="skip-vale">we're</span> working to release the following improvements soon:
-
-    -   Applying a coding standard by default to new repositories
-    -   Deleting a coding standard
+    This is a new Codacy feature and <span class="skip-vale">we're</span> continuing to improve it.
```

Comment:
Now that we already delivered the most important and planned improvements to the feature, I prefer not to be too specific about potential future iterations (otherwise we'll need to remember to update this notice whenever there our plans change).